package.json

```diff
@@ -45,7 +45,8 @@
   "dependencies": {
     "classnames": "^2.2.6",
     "fiori-fundamentals": "^1.4.0",
-    "focus-trap-react": "^6.0.0"
+    "focus-trap-react": "^6.0.0",
+    "prop-types": "^15.7.1"
```

Comment:
hrm. this can probably be a devDep.

Reply:
I don't think I agree with that. It needs to be exported in our built code so consumers receive prop-type warnings.

Reply:
https://github.com/facebook/prop-types#how-to-depend-on-this-package

Reply:
im sold. 🚢